src/test/java/model/FinancialIntegrationCurlCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FinancialIntegrationCurlCommandTest {
+    @Test
+    public void httpMethodConstructTest() {
+
+        FinancialIntegrationCurlCommand financialIntegrationCurlCommand = new FinancialIntegrationCurlCommand(
+                "GET",
+                "https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions",
+                true,
+                "application/json",
+                "json",
+                "Bearer $JWT",
+                "~/Downloads/fiResults.json");
+
+        assertEquals(financialIntegrationCurlCommand.getHttpMethod(),"GET");
+
+        assertEquals(financialIntegrationCurlCommand.getFinancialIntegrationBaseUrl(),
+                "https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions");
+
+        assertEquals(financialIntegrationCurlCommand.isIgnoreDocStatusQuery(), true);
+
+        assertEquals(financialIntegrationCurlCommand.getContentTypeHeader(), "application/json");
+
+        assertEquals(financialIntegrationCurlCommand.getDocumentFormatHeader(), "json");
+
+        assertEquals(financialIntegrationCurlCommand.getAuthorizationHeader(), "Bearer $JWT");
+
```

Comment:
The next step is to write a test that will fail until you write the code to make it pass. This test should be in its own method (@Test...public void testToUrl())

In this method, you need to build out a string that will represent the final output you expect from this class -- the final Curl command as you know it needs to end up being. You can use assert, even on a method that doesn't exist yet (toUrl).

Does that make sense?